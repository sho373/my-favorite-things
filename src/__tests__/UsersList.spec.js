import React from 'react';
import { shallow } from 'enzyme';
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Users from '../pages/users-list/users.component';

configure({adapter: new Adapter()});

it('should render UsersList page component', () => {
    const mockUser = {
        params:{
            displayName:"testUser"
        }
    };
    expect(shallow(<Users match={mockUser}/>)).toMatchSnapshot();
});