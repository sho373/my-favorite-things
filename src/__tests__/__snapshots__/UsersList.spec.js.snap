// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UsersList page component 1`] = `ShallowWrapper {}`;
